chartlets.js/packages/lib/src/actions/handleHostStoreChange.ts

```diff
@@ -91,6 +142,7 @@ function getHostStorePropertyRefs(): PropertyRef[] {
                 callbackIndex,
                 inputIndex,
                 property: formatObjPath(input.property),
+                id: `${contribPoint}-${contribIndex}-${callbackIndex}-${inputIndex}`,
```

Comment:
Do not store that value, instead compute it where it is needed, memoize if this has a performance impact.

Reply:
Only callback ID should be required:

```